tv_apps/browser/js/browser_dialog.js

```diff
@@ -31,13 +26,7 @@ var BrowserDialog = {
     };
 
     var elementIDs = [
-      'browser-dialog-base', 'browser-dialog',
-      'browser-dialog-title', 'browser-dialog-msg',
-      'browser-dialog-input',
-      'browser-dialog-input-area',
-      'browser-dialog-input-clear',
-      'browser-dialog-button',
-      'browser-dialog-button1', 'browser-dialog-button2',
+      'browser-dialog-base'
     ];
 
```

Comment:
I think we can directly declare variable here instead of loop through elementIDs for one element.